packages/validator/src/util/params.ts

```diff
@@ -216,9 +216,10 @@ function getSpecCriticalParams(localConfig: ChainConfig): Record<keyof ConfigWit
     // # DenebPreset
     /////////////////
     FIELD_ELEMENTS_PER_BLOB: denebForkRelevant,
-    MAX_BLOBS_PER_BLOCK: denebForkRelevant,
     MAX_BLOB_COMMITMENTS_PER_BLOCK: denebForkRelevant,
     KZG_COMMITMENT_INCLUSION_PROOF_DEPTH: denebForkRelevant,
+    MAX_BLOBS_PER_BLOCK: denebForkRelevant,
+    MAX_REQUEST_BLOB_SIDECARS: denebForkRelevant,
```

Comment:
`MAX_REQUEST_BLOB_SIDECARS` is part of network config, I don't think we need to check this from the vc side

We might rather wanna add it here
https://github.com/ChainSafe/lodestar/blob/99794d3b261bb50f9a6c7d9f6ca19291b3f30911/packages/validator/src/util/params.ts#L137-L138